cloud-functions/http_standardize_names/dev_requirements.txt

```diff
@@ -0,0 +1 @@
+pytest
```

Comment:
I was thinking the same thing.  We should move that file out of all the individual functions themselves.

Reply:
I was also wondering if it may make sense to move the requirements.txt as well to the same location as there may be some overlap between the different cloud functions.  Thoughts?  @DanPaseltiner

Reply:
Unfortunately we can't do that because GCP expects a `requirements.txt` files for Python runtime Cloud Functions in order to install the proper dependencies.